packages/admin/cms-admin/src/dam/DataGrid/thumbnail/DamThumbnail.tsx

```diff
@@ -66,15 +66,7 @@ export const DamThumbnail = ({ asset }: DamThumbnailProps) => {
             thumbnail = (
                 <>
                     <ImageThumbnail onMouseOver={handleMouseOver} onMouseLeave={handleMouseLeave} src={asset.image.thumbnailUrl} />
-                    <Popper
-                        sx={{ zIndex: 1301 }}
-                        open={open}
-                        anchorEl={anchorEl}
-                        placement="auto-end"
-                        onResize={undefined}
-                        onResizeCapture={undefined}
```

Comment:
Does this have anything to do with the update or was it unnecessary before?

(since I can't find any info relating to this change in the changelog or migration guide)

Reply:
It was only necessary due to a typescript error that has been fixed with MUI v6